src/CatalogGraphQLClient.php

```diff
@@ -83,23 +136,59 @@ public function find(array $searchBody, array $fields, int $page = 1, $perPage =
             $searchBody[SearchBody::FILTERS] = (new FilterHelper($searchBody[SearchBody::FILTERS]))->getSearchBodyFilters();
         }
 
-        $functionName = Types\Query::GET_PRODUCTS;
-
         $data = $this->call(
-            $functionName,
+            $type,
```

Comment:
I liked it better when it was named `$functionName` because the word "type" is very vague
But that's not a big deal